index.js

```diff
@@ -2613,6 +2613,42 @@
   }
   S.ifElse = def('ifElse', {}, [Pred(a), Fn(a, b), Fn(a, b), a, b], ifElse);
 
+  //# when :: (a -> Boolean) -> (a -> b) -> a -> b
```

Comment:
@JAForbes, on Gitter:

> I use `R.when` all the time, particularly with streams

Could you provide an example, James? Seeing a real use case would be helpful, I think.